Makefile

```diff
@@ -258,12 +258,15 @@ verify-extended: check-generate check format test-cov test-cov-clean test-integr
 # Rules for local environment                                       #
 #####################################################################
 
+kind-up kind-down gardener-up gardener-down register-local-env tear-down-local-env test-e2e-local-fast test-e2e-local: export KUBECONFIG = $(GARDENER_LOCAL_KUBECONFIG)
+
 kind-up: $(KIND)
 	mkdir -m 775 -p $(REPO_ROOT)/dev/local-backupbuckets
-	$(KIND) create cluster --name gardener-local --config $(REPO_ROOT)/example/gardener-local/kind/cluster-$(KIND_ENV).yaml --kubeconfig $(REPO_ROOT)/example/gardener-local/kind/kubeconfig
+	$(KIND) create cluster --name gardener-local --config $(REPO_ROOT)/example/gardener-local/kind/cluster-$(KIND_ENV).yaml --kubeconfig $(GARDENER_LOCAL_KUBECONFIG)
 	docker exec gardener-local-control-plane sh -c "sysctl fs.inotify.max_user_instances=8192" # workaround https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files
-	cp $(REPO_ROOT)/example/gardener-local/kind/kubeconfig $(REPO_ROOT)/example/provider-local/base/kubeconfig
-	$(LOCAL_GARDEN_KUBECONFIG) kubectl apply -k $(REPO_ROOT)/example/gardener-local/calico --server-side
+	cp $(GARDENER_LOCAL_KUBECONFIG) $(REPO_ROOT)/example/provider-local/base/kubeconfig
+	export
```

Comment:
Why is `export` needed? 
It is applying all my local envs to the target, but I don't see that any of them is required in this context. It is still working when I remove this statement.

Reply:
I think, `export` is needed when using recent versions of GNU make which doesn't export all variables to child processes by default, see `EXPORT_ALL_VARIABLES` in https://www.gnu.org/software/make/manual/html_node/Special-Targets.html and https://www.gnu.org/software/make/manual/html_node/Variables_002fRecursion.html#Variables_002fRecursion.
Old versions of GNU make and the default macOS make (I think), treat all variables as if they had the `export` directive.

On my system:
```
$ make --version
GNU Make 4.3
Built for x86_64-apple-darwin19.2.0
Copyright (C) 1988-2020 Free Software Foundation, Inc.
```

```make
foo: bar=foo
foo:
	printenv bar
```
yields
```
$ make foo
printenv bar
make: *** [Makefile:263: foo] Error 1
```

while
```make
foo: export bar=foo
foo:
	printenv bar
```
yields
```
$ make foo
printenv bar
foo
```

Can you confirm, that you are using the default macOS make? That would confirm my assumption.

Reply:
Yes, I'm using default `make` from macOS.
Don't we need the `export` statement in other targets which rely on local `KUBECONFIG` env like gardener-up etc. then too?

Reply:
My bad, I looked at the wrong line. Thought your comment was about line 261... 🙈